src/plugins/kbn_vislib_vis_types/public/sankey.js

```diff
@@ -0,0 +1,55 @@
+define(function (require) {
+  return function HistogramVisType(Private) {
+    var VislibVisType = Private(require('ui/vislib_vis_type/VislibVisType'));
+    var Schemas = Private(require('ui/Vis/Schemas'));
+    var sankeyBuilder = Private(require('ui/agg_response/sankey/sankey'));
+
+    return new VislibVisType({
+      name: 'sankey',
+      title: 'Sankey chart',
+      icon: 'fa-sankey-chart',
+      description: 'Sankey charts are ideal for displaying the parts of some whole. For example, sales percentages by department.' +
+       'Pro Tip: Sankey charts are best used sparingly, and with no more than 7 slices per sankey.',
+      params: {
+        defaults: {
+          shareYAxis: false,
+          isDonut: false
+        },
+        editor: require('plugins/kbn_vislib_vis_types/editors/sankey.html')
+      },
+      sankeyConverter: sankeyBuilder,
+      hierarchicalData: false,
+      schemas: new Schemas([
+        {
+          group: 'metrics',
+          name: 'metric',
+          title: 'Slice Size',
+          min: 1,
+          aggFilter: ['sum', 'count', 'cardinality'],
```

Comment:
Yes, we can. It just because the code base was copy from pie, and I used `bucket.doc_count` to process resp. The newest commit change to use `metric.getValue(b)` and now we can use `min/max/avg` here.